tests/fakes.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import, division, print_function
+
+from pyro.distributions.normal import Normal
+from pyro.distributions.random_primitive import RandomPrimitive
+
+
+class NonreparameterizedNormal(Normal):
+    reparameterized = False
+
+
+nonreparameterized_normal = RandomPrimitive(NonreparameterizedNormal)
```

Comment:
why do you need this? cant you set the class attribute directly in the tests in your conditional?

Reply:
Once all our distributions are migrated, are we planning to use this as an alias for `has_rsample`?